### ISSUE TITLE
installing command throws a compile error

### ISSUE THREAD

**Issue by LeonB** [itchyny/gojq#24]
```
~$ go get -u github.com/itchyny/gojq/cmd/gojq
# github.com/itchyny/gojq/cli
Workspaces/go/src/github.com/itchyny/gojq/cli/error.go:81:12: er.Position undefined (type participle.Error has no field or method Position)
Workspaces/go/src/github.com/itchyny/gojq/cli/error.go:86:47: er.Position undefined (type participle.Error has no field or method Position)
Workspaces/go/src/github.com/itchyny/gojq/cli/error.go:87:37: er.Position undefined (type participle.Error has no field or method Position)
Workspaces/go/src/github.com/itchyny/gojq/cli/error.go:91:45: er.Position undefined (type participle.Error has no field or method Position)
Workspaces/go/src/github.com/itchyny/gojq/cli/error.go:92:29: er.Position undefined (type participle.Error has no field or method Position)
```

**Comment by itchyny** [comment]
How about `GO111MODULE=on go get github.com/itchyny/gojq/cmd/gojq` ?

**Comment by LeonB** [comment]
Yep, that's it! 

Updating `alecthomas/participle` to the latest version also worked. Should I submit that as a pull request?

**Comment by itchyny** [comment]
I expect the user to enable Go modules on installing but also fixed the implementation in 7990d591a2c4573d39aa2815e189898e66fff8dd, closing.
